test/blackbox/common/mesh_event_handler.h

```diff
@@ -90,6 +90,20 @@ struct cond_flag {
 	bool flag;
 };
 
+typedef struct {
+	mesh_event_t  currentEv;
+	mesh_event_t  nextEv;
+} meshlink_node_status_lookup_t;
+
+typedef struct {
+	uint16_t                        node_id;
```

Comment:
I don't see anything using this member variable, so just remove it.

Reply:
Removed node_id variable from structure